discord/types/onboarding.py

```diff
@@ -56,9 +56,9 @@ class PromptOption(TypedDict):
     id: Snowflake
     channel_ids: SnowflakeList
     role_ids: SnowflakeList
-    emoji: NotRequired[Emoji]
-    emoji_id: NotRequired[Snowflake]
-    emoji_name: NotRequired[str]
-    emoji_animated: NotRequired[bool]
+    emoji: NotRequired[Emoji | None]
+    emoji_id: NotRequired[Snowflake | None]
+    emoji_name: NotRequired[str | None]
+    emoji_animated: NotRequired[bool | None]

```

Comment:
These are only supposed to be nullable (NotRequired) and not optional, see https://discord.com/developers/docs/resources/guild#guild-onboarding-object-prompt-option-structure

Reply:
> These are only supposed to be nullable (NotRequired) and not optional, see https://discord.com/developers/docs/resources/guild#guild-onboarding-object-prompt-option-structure

hmm description in that case it should also be different i think:

![image](https://github.com/Pycord-Development/pycord/assets/61660055/78a5b7dc-d72e-442c-b7e2-9bbeb9db0315)

Reply:
Yes, description is required yet nullable.

Reply:
@Dorukyum should be ok in recent commit, ive also included type of description as it was not correct